ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewHeaderFooterViewDemoController.swift

```diff
@@ -156,3 +163,48 @@ extension TableViewHeaderFooterViewDemoController: TableViewHeaderFooterViewDele
         return false
     }
 }
+
+extension TableViewHeaderFooterViewDemoController: DemoAppearanceDelegate {
+    func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return
+        }
+
+        fluentTheme.register(tokenSetType: TableViewHeaderFooterViewTokenSet.self,
+                             tokenSet: isOverrideEnabled ? themeWideOverrideTableViewCellTokens : nil)
+    }
+
+    func perControlOverrideDidChange(isOverrideEnabled: Bool) {
+        overrideTokens = isOverrideEnabled ? perControlOverrideTableViewCellTokens : nil
+        self.tableView.reloadData()
+    }
+
+    func isThemeWideOverrideApplied() -> Bool {
+        return self.view.window?.fluentTheme.tokens(for: TableViewHeaderFooterViewTokenSet.self) != nil
+    }
+
+    // MARK: - Custom tokens
+    private var themeWideOverrideTableViewCellTokens: [TableViewHeaderFooterViewTokenSet.Tokens: ControlTokenValue] {
+        return [
+            .textColor: .uiColor {
+                // "Grape"
+                return UIColor(light: GlobalTokens.sharedColor(.grape, .tint10),
+                               dark: GlobalTokens.sharedColor(.grape, .shade40))
+            }
+        ]
+    }
+
+    private var perControlOverrideTableViewCellTokens: [TableViewHeaderFooterViewTokenSet.Tokens: ControlTokenValue] {
```

Comment:
nit: Could these be changed to something like `perControlOverrideHeaderFooterTokens`?